### PR TITLE
cass-config-builder

### PR DESCRIPTION
**Pull request by developer-guy**
## New Image Pull Request Template

I didn't include any tests since the image throws an error as the upstream did.

### Image Size

- [x] The Image is smaller in size than its common public counterpart.
- [ ] The Image is larger in size than its common public counterpart (please explain in the notes).

Notes:

### Image Vulnerabilities

- [x] The Grype vulnerability scan returned 0 CVE(s).
- [] The Grype vulnerability scan returned > 0 CVE(s) (please explain in the notes).

Notes:

@dlorenc said they nacked the CVEs and it should be good tomorrow then.

### Image Tagging

- [ ] The image is _not_ tagged with version tags.
- [x] The image is tagged with :latest
- [ ] The image is not tagged with :latest (please explain in the notes).

Notes:

### Basic Testing - K8s cluster

- [x] The container image was successfully loaded into a kind cluster.
- [] The container image could not be loaded into a kind cluster (please explain in the notes).

Notes:

### Basic Testing - Package/Application

- [x] The application is accessible to the user/cluster/etc. after start-up.
- [ ] The application is not accessible to the user/cluster/etc. after start-up. (please explain in the notes).

Notes:

### Helm

- [x] A Helm chart has been provided and the container image can be used with the chart.  If needed, please add a -compat package to close any gaps with the public helm chart.
- [ ] A Helm chart has been provided and the container image is not working with the chart (please explain in the notes).
- [] A Helm chart was not provided.

Notes:

### Processor Architectures

- [ ] The image was built and tested for x86_64.
- [ ] The image could not be built for x86_64 (please explain in the notes).
- [x] The image was built and tested for aarch64.
- [ ] The image could not be built for aarch64. (please explain in the notes).

Notes:

### Functional Testing + Documentation

- [ ] Functional tests have been included and the tests are passing.  All tests have been documnted in the notes section.

Notes:

### Environment Testing + Documentation

- [ ] There has not been a request and/or there is no indication that this image needs tested on a public cloud provider.
- [ ] The container image has been tested successfully on a public cloud provider (AWS, GCP, Azure).
- [ ] The container image has not been tested successfully on a public cloud provider (AWS, GCP, Azure) (please explain in the notes).

Notes:

### Version

- [x] The package version is the latest version of the package.  The latest tag points to this version.
- [ ] The package version is the not the latest version of the package (please explain in the notes).

Notes:

### Dev Tag Availability

- [x] There is a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base')
- [ ] There is not a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base') (please explain in the notes).

Notes:

### Access Control + Authentication

- [x] The image runs as `nonroot` and GID/UID are set to 65532 or upstream default
- [ ] Alternatively the username and GID/UID may be a commonly used one from the ecosystem e.g: postgres
- [ ] The image requires a non-standard username or non-standard GID/UID (please explain in the notes).

### ENTRYPOINT

- [x] applications/servers/utilities set to call main program with no arguments e.g. [redis-server]
- [ ] applications/servers/utilities not set to call main program with no arguments e.g. [redis-server] (please explain in the notes)
- [ ] base images leave empty.
- [ ] base image and not empty (please explain in the notes).
- [ ] dev variants is set to entrypoint script that falls back to system.
- [ ] dev variants is not set to entrypoint script that falls back to system (please explain in the notes).

### CMD

- [ ] For server applications give arguments to start in daemon mode (may be empty)
- [ ] For utilities/tooling bring up help e.g. `–help`
- [ ] For base images with a shell, call it e.g. [/bin/sh]

### Environment Variables

- [ ] Environment variables added.
- [ ] Environment variables not added and not required.

### SIGTERM

- [ ] The image responds to SIGTERM (e.g., `docker kill $(docker run -d --rm cgr.dev/chainguard/nginx)`)

### Logs

- [ ] Error logs write to stderr and normal logs to stdout. Logs DO NOT write to file.

### Documentation - README

- [x] A README file has been provided and it follows the README template.

### CODE REVIEW

**Comment by imjasonh** [comment]
> I didn't include any tests since the image throws an error as the upstream did.

This isn't super reassuring. Is there anything we can test about the image to ensure even a minimum level of functionality?

Is there an upstream issue about why their image fails basic tests?

**Comment by developer-guy** [comment]
>Is there an upstream issue about why their image fails basic tests?

I don't think so maybe @mamccorm can help us with that?

> Is there anything we can test about the image to ensure even a minimum level of functionality?

To be honest I couldn't find anything since the image is failing already

**Comment by developer-guy** [comment]
according to the latest comments in the image request [here](https://github.com/chainguard-dev/image-requests/issues/519#issuecomment-1887917220) we understand that how they use this image so we created test cases to test the image we built to ensure whether it fits their needs. We used the same Helm chart to deploy cass-operator and edited the ImageConfig to ensure that the image we built is going to be used.

File: images/cass-config-builder/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+## Get It!
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/cass-config-builder:latest
+```
+<!--getting:end-->
+
+<!--body:start-->
+<!--body:end-->
```
Comment (mamccorm) [inline images/cass-config-builder/README.md:29]: No docs on how to use this image - we'd expect the usual here, i.e links to docs / helm chart with example, docker (don't think docker is applicable here). 

We've done this in the test but we should have some notes here on how to install the chart, where to find it, and example to run it (with arf to override our images)

**Review by mamccorm — APPROVED** [review]

**Pull request merged**